api/.env.template

```diff
@@ -44,6 +44,8 @@ LOTTERY_PUBLISH_PROCESSING_CRON_STRING=58 23 * * *
 LOTTERY_PROCESSING_CRON_STRING=0 * * * *
 # how many days till lottery data expires
 LOTTERY_DAYS_TILL_EXPIRY=45
+# allow partners and jadmins to create duplicate listings
+ALLOW_PARTNERS_TO_CREATE_DUPLICATES=FALSE
```

Comment:
can we change this to `ALLOW_PARTNERS_TO_DUPE_LISTINGS` or something like that, to make it clear that this is about duplicating listings